### PR TITLE
Scalar converter

### PR DESCRIPTION
**Pull request by jessdtate**
two modules: ConvertScalarToMatrix and ConvertMatrixToScalar.  Currently only implemented with Int32 scalars.  Once the scalar port is changed to allow both types, these modules will need to be changed to reflect that.

### CODE REVIEW

File: src/Modules/Math/ConvertScalarToMatrix.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+   DEALINGS IN THE SOFTWARE.
+*/
+
+#ifndef MODULES_MATH_CONVERTSCALARTOMATRIX_H
+#define MODULES_MATH_CONVERTSCALARTOMATRIX_H
+
+#include <Dataflow/Network/Module.h>
+#include <Modules/Legacy/Math/share.h>
```
Comment (dcwhite) [inline src/Modules/Math/ConvertScalarToMatrix.h]: Needs to be Modules/Math/share.h to match the library.

File: src/Modules/Math/ConvertMatrixToScalar.h [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+   DEALINGS IN THE SOFTWARE.
+*/
+
+#ifndef MODULES_MATH_CONVERTMATRIXTOSCALAR_H
+#define MODULES_MATH_CONVERTMATRIXTOSCALAR_H
+
+#include <Dataflow/Network/Module.h>
+#include <Modules/Legacy/Math/share.h>
```
Comment (dcwhite) [inline src/Modules/Math/ConvertMatrixToScalar.h]: Needs to be Modules/Math/share.h to match the library.

**Comment by jessdtate** [comment]
oops.  Fixed the share.h includes.

**Comment by dcwhite** [comment]
Thanks

**Pull request merged**